fitbenchmarking/fitbenchmarking
# Updating func and jac evaulation files

#### Description of Work

Fixes #624 

Updating NIST and CUTEst function and Jacobian evaluation files to test vector evaluation  


#### Testing Instructions

1. Check tests pass.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
